### PR TITLE
EDSC-971 Sort facets case-insensitively.

### PR DESCRIPTION
**Pull request by mightynimble**

### CODE REVIEW

**Comment by bilts** [comment]
You committed a screenshot. Please remove it.

File: spec/features/data_access/service_access_spec.rb [hunk 1]
```diff
@@ -77,8 +77,8 @@
             is_esi_completed = true if page.has_content?("Complete")
             sleep 0.5
             retries += 1
+            Rails.logger.info "-- Retried: #{retries}"
```
Comment (bilts) [inline spec/features/data_access/service_access_spec.rb]: This is almost certainly going to cause a merge conflict

**Comment by bilts** [comment]
This needs a test

File: spec/features/data_access/service_access_spec.rb (hunk 1 above)
Comment (mightynimble) [inline spec/features/data_access/service_access_spec.rb]: I will revert this change since you are working on it in another pr already.

**Comment by mightynimble** [comment]
I added a test in ```collection_facets_spec``` "shows facets in a case insensitive order". Is this what you are expecting?

**Pull request merged**